en/lessons/specifics/benchmarking.md

```diff
@@ -0,0 +1,346 @@
+---
+version: 1.0.0
+title: Benchmarking 
+redirect_from:
+  - /lessons/specifics/benchmarking/
+---
+
+We can't just guess about which functions are fast and which are slow - we need actual measurements when we're curious. That's where benchmarking comes in. In this lesson, we'll learn about how easy it is to measure the speed of our code.
```

Comment:
I think a bit more of a _why_ would be good. Like why do we benchmark, and when do we benchmark. Again, might be me being too much into this but I always find it important to tell people that benchmarking can be fun and that's good but in the end, they should benchmark what matters and that if they make a function 10 times faster that now takes 1 microsecond instead of 10 microseconds it might not be worth it if they spend 20 milliseconds in the database :) 

Or something to that tune :)

Reply:
Personally, I think one of the main values of Elixir School is the concise nature of the lessons. As a learning and reference text, I think our focus is in fact just the means of doing this in Elixir, not the nitty gritty of why to benchmark and how to do it right etc.

Reply:
Not that familiar with elixir school, if that's the spirit then I'm all for it and retract my statement. There's value in being concise and I'm not always best at finding it :grin:

Reply:
@PragTob when I kicked this off I did so because in general I find tech books to be _too_ verbose and I don't want to be reading paragraphs and paragraphs of thoughts.  With our lessons we've tried to be concise and focus on examples.  This isn't intended to the the end-all-be-all of Elixir resources but rather an opportunity to wet your appetite and hopefully encourage you to go out and continue to learn 😀